### PR TITLE
Updating dependabot file to support github actions, python and docker packages

### PR DESCRIPTION
**Pull request by rajpalc7**

### CODE REVIEW

**Comment by rajpalc7** [comment]
Hi @WadeBarnes , if you could review this that would be great.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -5,16 +5,67 @@ updates:
...
+  - package-ecosystem: "pip"
+    directory: "/test"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Canada/Pacific"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
```
Comment (WadeBarnes) [inline .github/dependabot.yml:38]: I don't see any dependency manifests in this folder.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -5,16 +5,67 @@ updates:
...
+  - package-ecosystem: "pip"
+    directory: "/tails_server"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Canada/Pacific"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
```
Comment (WadeBarnes) [inline .github/dependabot.yml:50]: I don't see any dependency manifests in this folder.

File: .github/dependabot.yml [hunk 3]
```diff
@@ -5,16 +5,67 @@ updates:
...
+  - package-ecosystem: "pip"
+    directory: "/tails_server/config"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Canada/Pacific"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
```
Comment (WadeBarnes) [inline .github/dependabot.yml:62]: I don't see any dependency manifests in this folder.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
@rajpalc7,  There are a number of update sections for folders that don't seem to contain any dependency manifests.  Am I missing something?

**Comment by rajpalc7** [comment]
Thanks @WadeBarnes I have made the necessary changes as per your suggestion

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**